src/Apps/Legion/Adapters/Standalone/DataManagement/Models/DataManagementAdapter.types.ts

```diff
@@ -60,13 +64,13 @@ export interface IUpsertTableAdapterParams {
 }
 
 export interface ITable {
-    Columns: Array<string>;
+    Columns: Array<ITableColumn>;
     Rows: Array<Array<any>>;
 }
 
 export interface ITableColumn {
-    column: string;
-    dataType: any;
+    columnName: string;
+    columnDataType: any;
```

Comment:
Is columnDataType able to be expressed as a string?

Reply:
ahh good point, will change it to:
| 'bool'
| 'datetime'
| 'dynamic'
| 'guid'
| 'int'
| 'long'
| 'real'
| 'string'
| 'timespan'
| 'decimal'